src/Illuminate/Database/Query/Grammars/Grammar.php

```diff
@@ -874,4 +874,21 @@ public function getOperators()
     {
         return $this->operators;
     }
+
+    /**
+     * Add an ESCAPE statement to SQL if required.
+     *
+     * @param  string  $sql
+     * @param  array  $where
+     *
+     * @return string
+     */
+    protected function escapeIfRequired(string $sql, array $where): string
```

Comment:
If you look closely at the source of Laravel you won't find any scalar type decl

Reply:
Updated!

Reply:
So what if there are no scalar types in the framework? L5.6+ requires at least PHP 7.1 - what's the point of that if we don't use the features it provides? At the very least when sending MRs with new features everything that can be type-hinted should be type-hinted.

Reply:
Probably best to make a discussion in laravel/internals, I know some people have fairly strong opinions on it

Reply:
Yes, my comment was mainly about the consistency. Personally I do favor them but Laravel has not (yet?) adopted them 🤷‍♀️